README.rst

```diff
@@ -76,6 +76,39 @@ standard Maven project structure for all of the source code files::
   ./src/main/java/co/cdap/guides/TopClientsService.java
   ./src/main/java/co/cdap/guides/TopNClientsReducer.java
 
+The ``pom.xml`` includes the Hadoop dependencies required to write a MapReduce program, namely:
+
+- ``hadoop-mapreduce-client-core`` for the MapReduce interfaces such as ``Mapper`` and ``Reducer``; and
+- ``hadoop-common`` for commonly-used Hadoop classes such as the various ``Writable`` classes that can be used as
+  map output types. 
+  
```

Comment:
Perhaps add:
```
In general, all ``hadoop-*`` dependencies should be included with a scope of “provided”, since they are provided by the platform. Depending on what your app is doing, you may have one or more of these ``hadoop-*`` artifacts in your project.
```